test/unit/init.test.js

```diff
@@ -0,0 +1,373 @@
+/* Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import EmailPassword from "../../recipe/emailpassword";
+import ThirdPartyPasswordless from "../../recipe/thirdpartypasswordless";
+import ThirdParty from "../../recipe/thirdparty";
+import ThirdPartyEmailPassword from "../../recipe/thirdpartyemailpassword";
+import EmailVerification from "../../recipe/emailverification";
+import Passwordless from "../../recipe/passwordless";
+import Session from "../../recipe/session";
+import Multitenancy from "../../recipe/multitenancy";
+
+import EmailPasswordRecipeCore from "../../recipe/emailpassword/recipe";
+import ThirdPartyPasswordlessRecipeCore from "../../recipe/thirdpartypasswordless/recipe";
+import ThirdPartyRecipeCore from "../../recipe/thirdparty/recipe";
+import ThirdPartyEmailPasswordRecipeCore from "../../recipe/thirdpartyemailpassword/recipe";
+import EmailVerificationRecipeCore from "../../recipe/emailverification/recipe";
+import PasswordlessRecipeCore from "../../recipe/passwordless/recipe";
+import SessionRecipeCore from "../../recipe/session/recipe";
+
+import SuperTokens from "../../lib/build/supertokens";
+import assert from "assert";
+
+describe("Init tests for recipes", function () {
```

Comment:
Is there any way we can make sure all exposed functions are tested here? Automating it would be even better (e.g.: listing keys from the index files and calling all functions).
Please add the relevant checklist item to PR template to make sure new functions/recipes are added here.

Reply:
Ignoring theses as discussed